android/KMEA/app/src/main/java/com/tavultesoft/kmea/data/Keyboard.java

```diff
@@ -144,6 +150,7 @@ protected void fromJSON(JSONObject installedObj) {
       this.isNewKeyboard = installedObj.getBoolean(KB_NEW_KEYBOARD_KEY);
       this.font = installedObj.getString(KB_FONT_KEY);
       this.oskFont = installedObj.getString(KB_OSK_FONT_KEY);
+      this.displayName = installedObj.getString(KB_DISPLAY_NAME_KEY);
```

Comment:
Well, in retrospect... we did miss something.  The version in #5349 uses `optString` here.